### DISCUSSION TITLE
Support of more devices, e.g. wireless mice

### DISCUSSION THREAD

**Discussion by theonlytechnohead** [Sapd/HeadsetControl#186] [Ideas]
Would there be any possibility of adding similar support for wireless mice like the G703 / 903?
I'd love to have at least battery reporting and LED on/off so I can stay as far away as possible from G HUB.
Also, thank you for this awesome utility, it is already amazing to have!

**Comment by Sapd** [comment]
As long as it's about such simple features - which don't require architectural changes, I don't see why not.

**Comment by giuseppeM99** [comment]
i use piper and solaar to configure my mouse and get the battery status (G703)
https://github.com/pwr-Solaar/Solaar
https://github.com/libratbag/piper
